modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -21,14 +21,13 @@ You can add an object storage bucket as a replication repository to the {mtc-ful
 * *AWS* for AWS S3, MCG, and generic S3 providers:
```

Comment:
@apinnick  MTC provides 4 types of  Storage provider, AWS S3, S3, GCP, Azure. So it looks not accurate. the S3 is for MCG and generic S3 providers.